ui/components/component-library/text-field/text-field.scss

```diff
@@ -1,5 +1,5 @@
 .mm-text-field {
-  --text-field-height: var(--size, 40px);
+  --text-field-height: var(--size, 48px);
```

Comment:
Not sure of the intention here but if we want to change the default size of the TextField we can do it in the default props.